src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+
+"activeModeText": {
+    "message": "Aktiven Modus einschalten? Verzögerung bis zum Aufräumen:",
+    "description": "Enable Active Mode? Delay Before Cleaning:"
+  },
+
+"minutesText": {
+    "message": "Minute(n)",
+    "description": "Minute(s)"
+  },
+
+"activeModeTooltipText": {
+    "message": "Wenn der aktive Modus eingeschaltet ist wird das Aufräumen der Cookies beim Schließen eines Tabs ausgeführt",
+    "description": "When active mode is enabled, cookie cleanup is triggered on tab close."
+  },
+
+"statLoggingText": {
+    "message": "Gesamtanzahl gelöschter Cookies protokollieren",
+    "description": "Log Total Number Of Cookies Deleted"
+  },
+
+"statLoggingTooltipText": {
+    "message": "Zählt die Anzahl der gelöschten Cookies, wird im Willkommensbildschirm angezeigt",
+    "description": "Counts the number of cookies deleted during this session and in total. This is shown in the Welcome Screen."
+  },
+
+"resetCounterText": {
+    "message": "Zähler zurücksetzen",
+    "description": "Reset Counter"
+  },
+
+"showNumberOfCookiesInIconText": {
+    "message": "Zeigt die Anzahl der Cookies einer Domain in der Symbolleiste",
+    "description": "Show Number of Cookies for that Domain"
+  },
+
+"showNumberOfCookiesInIconTooltipText": {
+    "message": "Zeigt die Anzahl der Cookies einer Domain im Tooltip des Symbols in der Symbolleiste",
+    "description": "Show Number of Cookies for that Domain"
+  },
+
+"notifyCookieCleanUpText": {
+    "message": "Zeigt eine Benachrichtungen nach dem Aufräumen der Cookies",
+    "description": "Show Notification After Cookie CleanUp"
+  },
+
+"notifyCookieCleanUpTooltipText": {
+    "message": "Zeigt wie viele Cookies für welche Domains gelöscht wurden.",
+    "description": "Shows how many cookies were cleaned and for which domain."
+  },
+
+"contextualIdentitiesEnabledText": {
+    "message": "Aktiviert die Unterstützung für Firefox' \"Contextual Identities\" (EXPERIMENTELL Firefox 53+)",
+    "description": "Enable Support for Firefox's Contextual Identities (EXPERIMENTAL Firefox 53+)"
+  },
+
+"contextualIdentitiesTooltipText": {
+    "message": "NICHT aktivieren solange du nicht Firefox 53+ nutzt. Fügt eine Whitelist für JEDEN Container hinzu, den du nutzt. Die Standard-Whitelist bleibt davon unabhängig.",
+    "description": "Do NOT Enable unless you are on Firefox 53+. This will add a different whitelist for EACH containers you use. Your default whitelist is separate from these."
+  },
+
+"saveText": {
+    "message": "Speichern",
+    "description": "Save"
+  },
+
+"cancelText": {
+    "message": "Abbrechen",
+    "description": "Cancel"
+  },
+
+"defaultSettingsText": {
+    "message": "Standardeinstellungen",
+    "description": "Default Settings"
+  },
+
+"defaultSettingsTooltipText": {
+    "message": "WARNUNG: Diese Aktion wird auch deine URLs zurücksetzen. Erstelle eine Sicherung!",
+    "description": "WARNING: This will also clear your URLS as well. So make a backup of them!"
```

Comment:
I think the term "URL" is confusing here, since this is talking about the whitelist, as far as I understand. Maybe this should be changed to "WARNING: This will clear your whitelist as well, make sure you have a backup!"?